### PR TITLE
[Refactor] #182 Change DictionaryMainView UI

### LINKED ISSUE #182: [Refactor] Update Dictionary Main View UI
# Structure
Add a squirrel background image, restructure

- Corresponds to the changed design

스크린샷 2022-08-06 오후 4 21 39

### PR DESCRIPTION
**Pull request by mijinc**
# Issue Number
🔒 Close #182 

## Changes

- Changed files
    - campair/campair/View/Dictionary/Main/DictionaryMainView.swift

## New features

- Did add dictionaryBackground Image
- Did change DictionaryMainView-category components' padding

## Review points

- I wonder why the padding value became negative(-)

## Questions

- How do we respond to devise changes?

## References

스크린샷 2022-08-10 오전 10 27 30

## Checklist

- [x] Is the branch you are merging on correct?
- [x] Do you comply with coding conventions?
- [x] Are there any changes not related to PR?
- [x] Has my code been self-reviewed?

### CODE REVIEW

**Review by Gobans — COMMENTED** [review]
Good! mijinchoi~!! thank for your effort, It look so good.
But, just one problem
image
Because of device screen size, even if you scroll down, still be obscured.
I recommend give some padding at bottom of LazyVGrid.

File: campair/campair/View/Dictionary/Main/DictionaryMainView.swift [hunk 1]
```diff
@@ -18,23 +18,25 @@ struct DictionaryMainView: View {
     var body: some View {
         NavigationView {
             VStack(spacing: 0) {
+                Image("dictionaryBackground")
+//                    .ignoresSafeArea()
```
[the remark this answers is not in the document]
Comment (mijinc) [reply campair/campair/View/Dictionary/Main/DictionaryMainView.swift:22]: Thank you for the comments, Goban!!
I'll try to cover all your comments!
Do I need to re-write this pull request? or create a new pull request?

**Review by Gobans — APPROVED** [review]
Did a great job!! Sorry for late review. LGTM!

**Review by rriver2 — APPROVED** [review]
Mijin Choi ! LGTM !! ♡

**Pull request merged**